src/App.js

```diff
@@ -14,11 +15,64 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selected: '-',
+  }
+
+  handler = (item) => {
```

Comment:
```suggestion
  selectHandler = (item) => {
```
Try to name your functions more meaningfully